src/main/target/BEEROTORF4/target.h

```diff
@@ -58,7 +58,7 @@
 #define MAX7456_DMA_IRQ_HANDLER_ID          DMA1_ST0_HANDLER
 
 #define USE_SDCARD
-
+#undef USE_DSHOT
```

Comment:
`BEEROTOF4` doesn't support DSHOT? I thought it was an `OmnibusF4` PR :)

Reply:
That was a leftover from earlier testing for firmware size changes when removing Dshot. Well spotted!